libs/server-sdk/include/launchdarkly/server_side/integrations/persistent_store_core.hpp

```diff
@@ -0,0 +1,212 @@
+#pragma once
```

Comment:
Likely I will have to move this file to common in a subsequent PR. Because the config will need to contain this interface.